.env.setup

```diff
@@ -1,22 +0,0 @@
-APP_ENV=local
-APP_DEBUG=true
-APP_KEY=,faO^ofvGeJ8|Z.q{mWp=^i}Cmzj+k&K
```

Comment:
It seems like you removed `.env.setup` (necessary for setup) rather than `env.setup`, which is the file you created earlier. Perhaps you can delete the one you created and then squash the commits?